Positioning1/style2.css

```
@@ -0,0 +1,27 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+div {
+    width: 150px;
+    height: 150px;
+}
+
+.yellow {
+    background-color: yellow;
+}
+
+.blue {
+    background-color: blue;
+    position: absolute;
+    top: 75px;
+    left: 75px;
+}
+
+.green {
+    position: absolute;
+    background-color: green;
+    top: 150px;
+    left: 150px;
+}
```

Este ok cum ai procedat aici dar, ca sa poti evita sa calculezi px ca sa fie cutia fix in centru, poti modifica un pic codul astfel:

```suggestion
.blue {
    background-color: blue;
    top: 50%;
    left: 50%;
}

.green {
    background-color: green;
    top: 50%;
    left: 50%;
}

<div class="yellow"> 
    <div class="blue">
        <div class="green">
        </div>
    </div>
</div>
```

Astfel, pentru ca elementul cu position absolut se positioneaza in functie de cel mai apropiat parinte cu position diferit de static, cutia din interior se va pozitiona in functie de unde este positionat parintele.